binaries/mbc.cpp

```diff
@@ -613,6 +622,9 @@ int main(int argc, char *argv[])
         exit(1);
     }
 
+    DB::SetLogFile(std::string(db_dir) + "/mabain.log");
```

Comment:
We should not set log file here. The problem is that db_dir is the data directory. And users may set the log file to a different directory than the data directory. By setting the log file here, it will automatically create a log file under the data directory.

Reply:
commented that line